index.js

```diff
@@ -12,34 +12,42 @@ console.log(fifaData);
 (d) Away Team goals for 2014 world cup final
 (e) Winner of 2014 world cup final */
 
+console.log(wcf['Home Team Wins']);
+console.log(wcf['Away team Wins']);
+console.log(wcf);
```

Comment:
You didn't finish completing Task 1. 
Also, where did you define "wcf" ? 

Think about which higher order functions you can use to `filter` through the data , compare the data available to the data you need to access, and then console log it (:

https://www.youtube.com/watch?v=rRgD1yVwIvE